{"code": " ValueError(\"Input data is not a DataFrame\")", "compilation_passed": true, "time": 0.0015697479248046875, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input df is not a DataFrame.\")", "compilation_passed": true, "time": 0.001531839370727539, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input data must be a pandas DataFrame.\")", "compilation_passed": true, "time": 0.0015130043029785156, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input df is not a DataFrame\")", "compilation_passed": true, "time": 0.0014362335205078125, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"The input is not a pandas DataFrame.\")", "compilation_passed": true, "time": 0.001444101333618164, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError('Input df must be a DataFrame.')", "compilation_passed": true, "time": 0.0014493465423583984, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError('Input data is not a DataFrame.')", "compilation_passed": true, "time": 0.0014328956604003906, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError", "compilation_passed": true, "time": 0.001428842544555664, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input must be a DataFrame\")", "compilation_passed": true, "time": 0.0014231204986572266, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input data must be a DataFrame.\")", "compilation_passed": true, "time": 0.0014371871948242188, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input is not a DataFrame\")", "compilation_passed": true, "time": 0.0014300346374511719, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError('Input should be a DataFrame')", "compilation_passed": true, "time": 0.001416921615600586, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError('Input must be a pandas DataFrame')", "compilation_passed": true, "time": 0.0014221668243408203, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError('Input df is not a DataFrame')", "compilation_passed": true, "time": 0.0014231204986572266, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input must be a pandas DataFrame\")", "compilation_passed": true, "time": 0.0014247894287109375, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input data is not a DataFrame.\")", "compilation_passed": true, "time": 0.0014400482177734375, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError('The input df is not a DataFrame')", "compilation_passed": true, "time": 0.0014328956604003906, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"The input df must be a pandas DataFrame.\")", "compilation_passed": true, "time": 0.0014281272888183594, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input must be a DataFrame.\")", "compilation_passed": true, "time": 0.0014262199401855469, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(f\"{df.__class__.__name__} is not a DataFrame.\")", "compilation_passed": true, "time": 0.001474142074584961, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input df is not a pandas DataFrame.\")", "compilation_passed": true, "time": 0.0014128684997558594, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
{"code": " ValueError(\"Input should be a DataFrame.\")", "compilation_passed": true, "time": 0.0014331340789794922, "tests_passed": false, "error": "partially initialized module 'pandas' has no attribute '_pandas_datetime_CAPI' (most likely due to a circular import)"}
